modules/http-client/test/index.test.ts

```diff
@@ -32,13 +32,12 @@ describe("constructor", () => {
   });
 });
 
-describe("PhenylHttpClient as EntityClient", () => {
+// TODO: implement me!
+describe("PhenylHttpClient as http client", () => {
   before(() => {
     server.listen(8080);
   });
 
-  // TODO: add tests to check that PhenylHttoClient works as an Entity Client
```

Comment:
EntityClient is memory-db's one, so check Entity Client should be located at memory-db.